app/src/main/java/cz/ackee/sample/interactor/IApiInteractor.kt

```diff
@@ -0,0 +1,17 @@
+package cz.ackee.sample.interactor
+
+import cz.ackee.ackroutine.core.OAuthCredentials
+import cz.ackee.sample.model.SampleItem
+import kotlinx.coroutines.Deferred
+
+/**
+ * Interactor for communicating with API
+ */
+interface IApiInteractor {
+
+    fun getData(): Deferred<List<SampleItem>>
```

Comment:
In our applications we are trying to have Deferred only on ApiDescription level and Interactor has "hard data" as in
`suspend fun getData():List<SampleItem>`. Does it make sense here to make it the same?

Reply:
i will take a look at it and push changes to more recent branch with workin example app